src/components/rangeSelect.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from "react"
+
+import Select from "./select"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
+  handleChangeMin: (value) => void
+  handleChangeMax: (value) => void
```

Comment:
We pass other props as objects. Can we align this:

```suggestion
  handleChange: {
    min: (value) => void
    max: (value) => void
  }
```